.eslintrc.js

```diff
@@ -36,22 +36,13 @@ module.exports = {
             "error",
             2
         ],
-        "linebreak-style": [
+        "no-console": [
             "error",
-            "windows"
-        ],
-        "quotes": [
-            "warn",
-            "single"
-        ],
-        "no-unused-vars": [
-            "warn",
-            "all"
-        ],
-        "no-constant-condition": [
-            "warn"
-        ],
-        "no-console": "off"
+                 {"allow":
```

Comment:
Nit: fix indentation here and in the following lines, { should be at the same level of the previous "error",